docs/false_positives.md

```diff
@@ -247,6 +266,41 @@ The result is easier to read for other developers who might not be familiar
 with the custom version of the function. The standard library functions also
 tend to be faster and more correct than custom solutions.
 
+### <a name="use-const"></a> Use const whenever possible
+
+It is useful to tell the analyzer when the state is not going to change.
+This will make the analysis more precise and the code more readable.
+Mark methods that are not going to change the state of the object as const.
+
+Also, avoid using global variables they are harder to reason about.
```

Comment:
perhaps: avoid using global variables **as/because** they are harder to reason about